src/document/DocumentAnnotator.ts

```diff
@@ -75,6 +100,24 @@ export default class DocumentAnnotator extends BaseAnnotator {
         }
     };
 
+    handleSelectionChange = debounce(
+        (): void => {
+            // Clear current selection immediately
+            this.store.dispatch(setSelectionAction(null));
+
+            // Wait Pdf.js textLayer enhancement for 300ms (they hardcode this magic number)
+            this.selectionChangeTimer = window.setTimeout(() => {
+                this.store.dispatch(setSelectionAction(getSelectionItem()));
+                this.selectionChangeTimer = null;
+            }, TEXT_LAYER_ENHANCEMENT);
+        },
+        TEXT_LAYER_ENHANCEMENT,
+        {
+            leading: true,
```

Comment:
Are these options different than the default options? If so, can we add a comment explaining why?